examples/advanced_behaviors/alert/behavior.xml

```diff
@@ -139,8 +139,8 @@
                     alert:title="Choose an action"
                     trigger="press"
                     xmlns:alert="https://hyperview.org/hyperview-alert"
```

Comment:
Why are these changing from `hyperview.org` to `instawork.com`?

Reply:
The namespace distinguishes features that are supported in the core Hyperview client vs the ones that are custom extensions included in our apps. Some features get moved from our custom extensions to the core once they mature. For example, `alert` and `share` behaviors use the `hyperview.org` namespace. Things like `intercom`, `amplitude`, `redux` will likely always use the `instawork.com` namespace.

The one outlier is `share`, we didn't change the namespace immediately since it was in wide use, but there's a task to migrate to the `hyperview.org` namespace.